packages-new/components/menu/src/components/osds-menu/core/controller.spec.ts

```diff
@@ -0,0 +1,311 @@
+import {
+  OdsClearLoggerSpy,
+  OdsInitializeLoggerSpy,
+  OdsLoggerSpyReferences
+} from '@ovhcloud/ods-testing/src';
+
+import { Ods, OdsLogger } from '@ovhcloud/ods-common-core';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import { OdsMenu } from '../ods-menu';
+import { OdsMenuController } from './controller';
+import { OcdkSurface, OcdkSurfaceMock } from '@ovhcloud/ods-cdk';
+import { OdsMenuItem } from '../menu-item/ods-menu-item';
+
+class OdsMenuMock implements OdsMenu {
```

Comment:
use extends instead and put only what is actually needed (see how other components are doing it)